chemdash.py

```diff
@@ -216,33 +218,17 @@ def get_umap_graph():
 
     return umap_graph
 
-def get_columns_dropdown():
+
+def get_columns_dropdown(df: pd.DataFrame):
     numerics = ['int16', 'int32', 'int64', 'float16', 'float32', 'float64']
     all_cols = df.select_dtypes(include=numerics).columns
-    all_cols
     cols_to_remove = ['tsne_x', 'tsne_y', 'lap_x', 'lap_y', 'id', 'umap_x', 'umap_y',]
     cols_to_keep = [x for x in all_cols if x not in cols_to_remove]
     cols_to_keep.sort()
     return cols_to_keep
 
 
-
-#read in file with smiles and compound_id (optional)
-df = pd.read_csv("dataset_800.csv")
-#df = pd.read_csv("dataset_10k.csv")
-#df = df.head(2000)
-
-print ('There are ' + str(df.shape[0]) + ' compounds in your dataset')
-if all(x in df.columns for x in ['smiles', 'Compound_id']) and (df.shape[0]<10001):
-
-    # create asset directory for mol image files
-    if not os.path.isdir("assets/mol_images"):
-        os.mkdir('assets/mol_images')
-
-    df = prepare_data(df)
-    df['id'] = df.index
-
-
+def create_chemdash_app(df: pd.DataFrame):
     app = dash.Dash(__name__, assets_folder='assets')
```

Comment:
Currently the app exepcts the dataframe to exist.  In the next PR I'll let the user pick a file so the UI can come up quickly.

For production we'll probably want a queue instead of long-running UI callbacks.

Reply:
do you think we should bootstrap with a small df? I have noticed that sometimes plotly dash acts whacko without a placeholder.  I guess we can try a few things as we move forward